apps/academy-wep/components/learn/resource-card.tsx

```
@@ -1,119 +1,129 @@
+			<div className="relative z-10 flex flex-col h-full">
+				<div className="h-48 bg-gray-100 relative overflow-hidden flex-shrink-0">
+					<Image
+						src={resource.image || '/placeholder.svg'}
+						alt={resource.title}
+						className="w-full h-full object-cover transition-transform duration-700 group-hover:scale-105"
+						width={400}
+						height={240}
+					/>
 
-          <div className="absolute top-0 left-0 w-full h-full bg-gradient-to-t from-black/30 to-transparent opacity-0 group-hover:opacity-100 transition-opacity duration-500"></div>
+					<div className="absolute top-0 left-0 w-full h-full bg-gradient-to-t from-black/30 to-transparent opacity-0 group-hover:opacity-100 transition-opacity duration-500"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)